dissect/target/filesystem.py

```diff
@@ -1405,11 +1473,11 @@ def _get_from_entry(self, path: str, entry: FilesystemEntry) -> FilesystemEntry:
 class EntryList(list):
     """Wrapper list for filesystem entries.
 
-    Expose a getattr on a list of items. Useful in cases where
-    there's a virtual filesystem entry as well as a real one.
+    Expose a getattr on a list of items. Useful to access internal objects from specific filesystem implementations.
```

Comment:
```suggestion
    Expose a :func:`getattr` on a list of items. Useful to access internal objects from specific filesystem implementations.
```

Reply:
I'm not sure it works like that. `:func:` and friends are to deeplink to functions and classes within your code base. This is a Python native function. This would require `sphinx.ext.intersphinx` (https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) which we don't have configured for our docs.

Reply:
You are right indeed. In that case \`\`getattr\`\` should do it instead. While playing around a little I noticed that :func:\`getattr\` renders to `getattr()`, with added parenthesises, but obviously no deeplink. It might be an idea to stick with the :func: notation, if we want to ever add the Intersphinx extention in the future.

Reply:
Went in a different direction.